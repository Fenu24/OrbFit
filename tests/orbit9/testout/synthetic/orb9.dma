   2 1         3        
t0=   2.4588005000000000D+06 ;  from catalogue
coosys='EQU HEL INVL1B RAD';ilce=   2
  a(au)      h      k      p      q     lambda  nrev  gam                                           
  0.0000000000000000D+00
  0.2769719811013878D+01  0.3754964791233676D-01 -0.6682555642598412D-01  0.7679701325442391D-01  0.2449963791836228D-01  0.4720961418824407D+01         0  0.0000D+00
  0.2668700199923353D+01  0.2085258106203616D+00  0.1500861834070681D+00  0.1371769744737286D-01 -0.1071693876017957D+00  0.2346137011367259D+01         0  0.0000D+00
   1.52321629883272       0.255311228345412        1.54622391696371     
  5.522774206981330E-002   1.50821138710078        1.97432714894858     
  0.744612833727931       0.318546147667965       0.507160399600230     
   2.30604001072576       0.355581248616880        2.52580486826585     
 -0.2000000000000000D+06
  0.2769270619237487D+01  0.9225139059053240D-01  0.5399685644084880D-01  0.6844547044872723D-01 -0.3837039868074439D-01  0.1774797017432882D+01    -43440  0.1022D+02
  0.2667230474136566D+01  0.9082390771207330D-01 -0.2601625777883644D+00 -0.5501782636447760D-01  0.1199946287791006D+00  0.1319946557666741D+01    -45850  0.1322D+02
   49603.6856063284        14417.3285157538       -7961.10982348566     
  -27351.7600733674        71976.9227214383       -1821.35272838745     
   955114.702399169       -535715.125040601       -23265.2553324283     
   1048794.38250417        1013663.87935323        364967.083951319     
 -0.4000000000000000D+06
  0.2766954608320774D+01 -0.1328976213442958D+00  0.2906612893580910D-01 -0.6350364256033592D-02 -0.7790397055573216D-01  0.1294546672284714D+01    -86881  0.1060D+02
  0.2667080839263419D+01 -0.2427183406820465D+00 -0.4724846463596583D-01  0.8059767869757360D-01 -0.7510428463110773D-01  0.2676461721703023D+00    -91700  0.1434D+02
  -82253.7097541132        18241.0680916298       -3918.03531418040     
  -17221.8436065412       -100793.634709144        15558.3344668099     
   1527814.40497831       -3138334.21879392        226113.820306671     
   3452846.51379988        2612752.35027045       -960591.245703296     
 -0.6000000000000000D+06
  0.2767569851491973D+01  0.5796681558121727D-01 -0.6452522538024758D-01 -0.2917325773137519D-01 -0.7348813824375604D-01  0.2846251467472155D+01   -130322  0.1166D+02
  0.2668619425658634D+01 -0.3202509288813201D-01  0.2666614352329088D+00 -0.1027734596366617D+00  0.4814128639882978D-01  0.5151600771458286D+01   -137551  0.1412D+02
   56172.1979082155        210053.727547198       -27746.2934770369     
  -316232.527079942        66343.6517501357       -28299.9977708050     
  -1.41732934868075      -0.235714211654642      -0.317718332262037     
 -0.174567645809834       -2.29122505912115      -0.259218667019124     
 -0.8000000000000000D+06
  0.2767897580402806D+01  0.6043419621997238D-01  0.6508584951688874D-01 -0.6170162953154237D-01 -0.5071743775074360D-01  0.1396615793966559D+00   -173762  0.1046D+02
  0.2667300911451885D+01  0.2441692018376636D+00 -0.5363372664877366D-01  0.1267310541604919D+00 -0.1794498402470629D-01  0.3408616022697966D+01   -183401  0.1273D+02
  -4932.62988555885        65908.1617433566       -7306.59086650811     
  -96695.9929725514       -3512.11344219474       -11555.9550211558     
  0.185284047204731      -0.351341272506322      -3.555857723699647E-002
  0.382409804865889       0.361868335343080      -0.113018469949228     
 -0.1000000000000000D+07
  0.2767867938771931D+01 -0.1460016405704660D+00  0.6660147511017368D-02 -0.6036390734902758D-01  0.5744122303637979D-01  0.4099456509817594D+01   -217203  0.1189D+02
  0.2669925152732677D+01 -0.3032524193132172D-01 -0.2069820472623308D+00 -0.1041036114262424D+00 -0.2451223456576114D-01  0.1620699889237249D+01   -229251  0.1382D+02
   203482.830441922       -178959.516167559        4016.55931073114     
   299639.134794549        280218.890844795        68765.6215920186     
  -1.14670543103982       0.164530612430455      -0.251484942432022     
 -0.541384883702622       -1.52037683796329      -3.862427025741993E-002
 -0.1200000000000000D+07
  0.2767183384736431D+01  0.8319724912073451D-01 -0.7309572294804552D-01  0.1821810220511916D-01  0.8362418224970040D-01  0.3802252442847478D+01   -260644  0.1261D+02
  0.2670932050795493D+01 -0.2241582569246693D+00  0.5771194256166108D-01  0.1067578588481455D+00  0.5808491590686078D-01  0.6968658315264445D+00   -275101  0.1310D+02
   412313.204661439       -420312.161576427       -85720.6924891256     
   513572.624563633        616826.061829768        85180.1805352262     
 -0.443936367913774       0.377154779741018       0.142731219694677     
 -0.374447796479374      -0.668242293811885       2.333064658362981E-003
 -0.1400000000000000D+07
  0.2767208112758220D+01  0.3411339178923409D-01  0.6922840932737911D-01  0.4316366213586777D-01  0.7376954362794266D-01  0.1670146758274289D+01   -304085  0.1219D+02
  0.2667516619493025D+01  0.8472230637224638D-01  0.1737836142807397D+00 -0.7785140668151763D-01 -0.9018610288053218D-01  0.5471387719001533D+01   -320952  0.1212D+02
   380580.460409833        68888.3371315605       -22762.3516281238     
  -129198.096587849        518372.368427098        88015.2807838518     
  0.173241101285682       0.111369713292938       7.750049419974351E-003
 -0.104716404892863       0.277284405529926      -6.972237167906623E-002
 -0.1600000000000000D+07
  0.2770286060618806D+01 -0.1377926835850956D+00 -0.1174136467746653D-01  0.8961120904958637D-01  0.8493092468117653D-02  0.1048856659650847D+01   -347525  0.1276D+02
  0.2671330568456734D+01  0.2009049795170194D+00 -0.4239378311748689D-01  0.3702540477334450D-01  0.1021121561161855D+00  0.3656130634337741D+01   -366802  0.1336D+02
   643063.424602145       -333424.764369722       -122062.668407808     
   361540.476684945        813991.120620525       -51307.9283292770     
 -0.476727348320687       0.570355278788789       0.151890389562172     
 -0.648960449013501      -0.751860949513035      -0.106031960413352     
 -0.1800000000000000D+07
  0.2770428552066483D+01  0.1012063674834803D+00 -0.6607213261559326D-01  0.4466922203726713D-01 -0.7858941288725602D-01  0.4514823372410995D+01   -390966  0.1313D+02
  0.2670867808745169D+01 -0.9892553930448955D-01 -0.1947458304377772D+00 -0.1389625520395445D-01 -0.1247459150123843D+00  0.2555190606110775D+01   -412652  0.1505D+02
   1042974.12501335       -134992.791082081       -72636.0898745773     
   51307.0411213739        1298820.88380640       -211257.213837220     
   2.72457394062903        2.55098790927798      -0.569385118861870     
  -3.07802689924007        4.77435629989752       -1.29517502143921     
 -0.2000000000000000D+07
  0.2766027501373225D+01  0.1600537122325917D-01  0.6843697329952417D-01  0.1972958349350237D-01 -0.8686193408219280D-01  0.3047368034177813D+01   -434406  0.1379D+02
  0.2671744753519367D+01 -0.1836200423399891D+00  0.7986262241892494D-01 -0.3239439809627480D-01  0.1063320791658706D+00  0.1545438932194639D+01   -458502  0.1630D+02
  -141257.557784411       -1970353.74362852        349904.230391133     
   2567409.22854990       -143041.092597311       -77839.0867844927     
  -14.9120296857711      -0.469731711669066       -1.08615943112727     
   2.13834454100067       -17.9499999287749       -3.73618414365801     
