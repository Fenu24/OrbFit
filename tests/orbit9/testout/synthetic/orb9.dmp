   4 JUPITER   SATURN    URANUS    NEPTUNE  
t0=   2.4588005000000000D+06 ;  JPL DE 431 13 NOV 2019 0.0000
coosys='EQU HEL INVL1B RAD';ilce=   0
  a(au)      h      k      p      q     lambda  nrev  gam                                           
; planetary inverse masses as used
  9.9999402303776952D-01  1.0473486254633374D+03  3.4979017677866345D+03
  2.2902981613087039D+04  1.9412259775973071D+04
  0.0000000000000000D+00
  0.5203408675427578D+01  0.8863519227128651D-02  0.4791346061737496D-01 -0.2073322602311020D-02  0.1964169016143707D-02  0.4777294425153240D+01         0
  0.9580546104808700D+01  0.5091233617815853D-01  0.1634891025148407D-02  0.7015875372486742D-02 -0.4093518492527722D-02  0.5047217814115163D+01         0
  0.1916922931803378D+02  0.8454446117725107D-02 -0.4610514634968570D-01 -0.7024981819952272D-02  0.5586230684003522D-02  0.6040871417844688D+00         0
  0.3018062155199051D+02  0.2560497517658895D-02  0.9231460377627090D-02 -0.1229861147727340D-02 -0.6216022858118756D-02  0.6011866011871723D+01         0
 -0.2000000000000000D+06
  0.5202606250485768D+01  0.2107971838951457D-01 -0.5341195513569881D-01 -0.1650061702379190D-02  0.3201835941090767D-02  0.1961809288368749D+01    -16860
  0.9587627653800787D+01  0.1473586991063766D-01  0.1649096237577115D-01  0.4480291078860976D-02 -0.6142777699605386D-02  0.2357230752666241D+01     -6789
  0.1913171091182300D+02 -0.3658335974874748D-01  0.5139771342521594D-01  0.6795858361963046D-02 -0.4362600492709227D-02  0.4630332599377105D+01     -2381
  0.3020927869408359D+02  0.1343902818846670D-01  0.3955378179582306D-02 -0.5904833179920826D-02 -0.2541194630960221D-02  0.1000647753997502D+01     -1213
 -0.4000000000000000D+06
  0.5202186753251238D+01 -0.3047938028741962D-01  0.1475668194448173D-01  0.4775410079226981D-03  0.2873563721248563D-02  0.5447562067210844D+01    -33721
  0.9589970704742209D+01 -0.7133600530239059D-01 -0.2997735058046120D-01  0.2254167920320481D-02 -0.7984369044066490D-02  0.5907398164178435D+01    -13579
  0.1911964715934859D+02  0.5622128519495771D-01 -0.1557239716740955D-01 -0.8654272653225575D-02  0.3404191270963144D-02  0.2258925281757969D+01     -4761
  0.2992288014239060D+02 -0.8271427979213958D-02  0.1494340968601576D-01 -0.4870094534831171D-02  0.4110078789228710D-03  0.2332654548987212D+01     -2427
 -0.6000000000000000D+06
  0.5202955465669052D+01  0.1074354746496859D-01  0.3778242498411200D-01  0.7038387772957561D-03  0.2715513585896742D-02  0.2684830506388891D+01    -50581
  0.9583415802662822D+01  0.6979090542887555D-01  0.1471453983770986D-01 -0.1886875417055659D-02 -0.8006429919201698D-02  0.3086647469628458D+01    -20368
  0.1912643022314576D+02 -0.4647386120795984D-01 -0.9653715772266654D-02  0.8220475098959986D-02 -0.5569801841237216D-03  0.6275894410208275D+01     -7142
  0.3023510262850573D+02 -0.3029967429802971D-02  0.5043065974270511D-03 -0.5083021015545259D-02  0.4340902795257045D-02  0.3604176950285879D+01     -3641
 -0.8000000000000000D+06
  0.5202983078035690D+01  0.2148596988697329D-01 -0.5644043588413584D-01  0.2431207178807742D-02  0.1987362639460308D-02  0.6156999515401518D+01    -67442
  0.9581358075161756D+01 -0.1303277109415960D-01  0.9550436389468751D-02 -0.4261509719857443D-02 -0.6787355102778951D-02  0.3773563197226618D+00    -27157
  0.1916620765176388D+02  0.4779798216470274D-02  0.9859265214316137D-02 -0.8510586948543388D-02 -0.1224011671943345D-03  0.4121002082458650D+01     -9522
  0.3014091287087681D+02 -0.1276029149082060D-01  0.5906124280193807D-02  0.4294634949122320D-03  0.5925340579634607D-02  0.4833901223620303D+01     -4855
 -0.1000000000000000D+07
  0.5202470884494629D+01 -0.3867304865991442D-01  0.3009354068059276D-01  0.1984982566361326D-02  0.8659584704568956D-03  0.3333119758340928D+01    -84302
  0.9584485153456026D+01 -0.4759601780291648D-01 -0.3648089619101839D-01 -0.7486057484746009D-02 -0.4494823074524398D-02  0.3981379232417740D+01    -33947
  0.1923266629205002D+02  0.2734977188054213D-01  0.1551556132744901D-01  0.8834324393431126D-02  0.3936877182513981D-02  0.1974350675571649D+01    -11902
  0.2996068006747914D+02 -0.9440737013415121D-02 -0.1121866975399757D-01  0.2149798745368997D-02  0.4603083207590183D-02  0.6045552514839101D+01     -6069
 -0.1200000000000000D+07
  0.5202611508313023D+01  0.2001253835092846D-01  0.1987494662595056D-01  0.2986325452211975D-02  0.4155795902103625D-03  0.5744334845359702D+00   -101162
  0.9590676397245140D+01  0.7753651339197394D-01  0.3696506474984819D-01 -0.8046453367219620D-02 -0.7807113119272420D-03  0.1147297395947547D+01    -40736
  0.1927425011278281D+02 -0.2737473785021710D-01 -0.4679228105739545D-01 -0.6394780483024638D-02 -0.5134262959803584D-02  0.6052769298845366D+01    -14283
  0.3028523341115054D+02  0.2327345478164308D-02  0.5010910106604991D-03  0.6416824619034363D-02  0.2591410450824847D-02  0.1004263869768535D+01     -7282
 -0.1400000000000000D+07
  0.5203140672098982D+01  0.1541286140761312D-01 -0.5052327256626554D-01  0.2132317880240384D-02 -0.1137419099304041D-02  0.4074714950261142D+01   -118023
  0.9584319174796045D+01 -0.4153894993970932D-01 -0.6785081160098651D-02 -0.8262041714359731D-02  0.2206099509433225D-02  0.4661162538104707D+01    -47526
  0.1929804655596447D+02 -0.8322746125415262D-02  0.5913261907436502D-01  0.6872098691630037D-02  0.6660939894894012D-02  0.3693860655855949D+01    -16663
  0.3004553720684627D+02  0.6417158589217253D-03 -0.1580407010943207D-01  0.4787716349300497D-02 -0.2791636279850186D-02  0.2340328182984846D+01     -8496
 -0.1600000000000000D+07
  0.5203261087432159D+01 -0.3998126861157631D-01  0.3483434979760210D-01  0.2352797323793528D-02 -0.1104273314003807D-02  0.1257469501102610D+01   -134883
  0.9576617866886480D+01 -0.1896006520108820D-01 -0.3121539567837986D-01 -0.6143459774254087D-02  0.5836489056770172D-02  0.1976521399526655D+01    -54315
  0.1931278062704887D+02  0.3998282145957116D-01 -0.4794547194010609D-01 -0.3731385373079289D-02 -0.8755664930968757D-02  0.1384437537781084D+01    -19043
  0.3001364585156177D+02  0.7883520496024449D-02  0.4247794603904649D-03  0.3640282705394301D-02 -0.3939108227504232D-02  0.3624887817478905D+01     -9710
 -0.1800000000000000D+07
  0.5203249559653439D+01  0.2868978024600514D-01  0.1098884737954574D-01  0.1381153771825964D-02 -0.2698196312507075D-02  0.4768705128338157D+01   -151744
  0.9576491598765919D+01  0.6249556960750847D-01  0.4893313014308953D-01 -0.3626713625957389D-02  0.7111580551207812D-02  0.5457484745637315D+01    -61105
  0.1930178254093977D+02 -0.2998372033029944D-01  0.1454291553958813D-01  0.2193700261158000D-02  0.7883381903358869D-02  0.5445395942327499D+01    -21424
  0.3028251317168478D+02  0.2703724442241619D-02 -0.2104034014580286D-02 -0.8152315822413493D-03 -0.6853685584127401D-02  0.4883149109134063D+01    -10924
 -0.2000000000000000D+07
  0.5202506627019440D+01  0.8026499708995877D-02 -0.4903074588333620D-01  0.5953570913136278D-03 -0.2256701489516416D-02  0.2002132410181397D+01   -168604
  0.9584616644148518D+01 -0.5079517823534467D-01 -0.2598619766588204D-01 -0.1701817956779064D-03  0.8617893807000435D-02  0.2635633899144305D+01    -67894
  0.1925226977803903D+02 -0.1653396849944970D-02  0.9205797880003971D-02  0.8869259465909236D-04 -0.9601996065301443D-02  0.3303735315086256D+01    -23804
  0.2999084802205923D+02  0.1138930655347239D-01  0.4145448617638299D-02 -0.4108272356319748D-02 -0.3162001438713749D-02  0.6107042796770304D+01    -12138
