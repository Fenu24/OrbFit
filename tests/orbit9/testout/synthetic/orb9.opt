;input control and options file for orbit8
; 1. input file names
inplan= 'planxx.inc' ; input file name for initial conditions of the planets
ibar= 1 ; barycentric correction 1=yes 0=no
inbar='barsunxx.inc'; file containing the barycentric correction
inast='chunk.cat'; input file name for initial conditions of the asteroids
; 2. job description: ast. are ordered in two list, with and without LCE
nvz= 2 ; no. Lyapounov exponents
 1
 3

; 3. options: output           $ = option not yet implemented
dt= -200 ; time between two outputs
nout= 10000 ; output number at job termination
idump= 1000; number of outputs between dump/renormalisation
nsamp= 100; sampling ratio of the filter; choose 5, 20,50 or 100
iprqua= 2 ; how much output: 1 elem. filtr; 2 also sampl.;
sysz= 'HEL' ; output system: BAR, HEL, JAC, HEC
refz= 'INVL1B' ; output reference system: INVL1B, ECLM50, EQUM00, etc
; 4. options: variational equation
v1=  1.00d+06; variation vector norm forcing renormalisation
semim=    3.0; approx. semimajor axis for norm of var.vect.
; 5. options: propagator
h=  0.2; stepsize for multistep (maximum stepsize if automatic)
iauto= 1; automatic stepsize control 1=on 0=off
error= 1.d-13; target integration error, radiants/revolution^2
iord=  12;  order of multistep predictor
epms= 1.0d-12; convergence control for corrector $
iork=   12;  order of starter
eprk= 1.00d-10; convergence control for implicit Runge-Kutta
lit1= 10; iterations for first step
lit2= 4; iterations for following steps
imet= 1; main integration method: 1 multistep 2  symplectic
iusci=10; output of numerical convergence controls 0 no
icha=0; what to do if non convergent 0 interactive 1 batch, change step $
ll=12; control for Everhart (>0 automatic stepsize control)
; 6. control of close approach output
dmint=0.1; close appr. distance for terrestrial planets
dminj=1.0; close approach distance for giant planets
npoint=10; minimum number of data points for a deep close appr.
; 7. optional perturbations
irelj2=0; relativistic and J2 perturbations 0=no 1=yes
iyark=0; Yarkovsky effect as secular drift in a 0=no 3=yes 1,2=not in use

; END OF INPUT
