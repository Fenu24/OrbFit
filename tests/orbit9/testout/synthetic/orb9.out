; orbit9 vers. c, output file
 in input            4  planets: KEP HEL ECLM00
barycenter corr.flag  1 no. inner pla.   4
; planetary inverse masses as used
  9.9999402303776952D-01  1.0473486254633374D+03  3.4979017677866345D+03
  2.2902981613087039D+04  1.9412259775973071D+04
  planetary initial conditions as read
   2  asteroids required,   2  with LCE; numbers:
  1          3        
  asteroid 1          OK
  asteroid 3          OK
  file end looking for 20                  record           21
  Selection of the best stepsize, maximum= -0.200000000000000     
 orbit no.  1 sugg.step= 0.1216 a,e= 5.203  0.0487
 orbit no.  2 sugg.step= 0.2000 a,e= 9.581  0.0509
 orbit no.  3 sugg.step= 0.2000 a,e=19.169  0.0469
 orbit no.  4 sugg.step= 0.2000 a,e=30.181  0.0096
 orbit no.  5 sugg.step= 0.0291 a,e= 2.770  0.1776
 orbit no.  6 sugg.step= 0.0219 a,e= 2.669  0.3341
 min. step required  0.021858 selected -0.021739 nn=    92
  input complete, starting from t=  0.000000000000000E+000
 renormalisation var.eq.no    2
 time=    -400000.000000 norm=    1.68946D+06 gamma=    1.43399D+01
 end orbit8v, time =      -2000000.0000
